semana9/Projeto/astromatch-app/src/App.js

```diff
@@ -1,65 +1,65 @@
-import React, {useState , useEffect} from 'react';
-import axios from 'axios'
-import AstroProfiles from './components/AstroProfiles/AstroProfiles'
-import AstroMatches from './components/AstroMatches/AstroMatches'
-import { AstroHomeContainer , AstroButton , AstroAppIcons , AstroMatcheLogo } from './components/GlobalStyles/AstroStyles';
-import matchesIcon from './components/assets/metchesicon.png'
-import resetIcon from './components/assets/reset.png'
-import AstrometcheLogo from './components/assets/astromatchelogo.png'
+import React, { useState , useEffect } from 'react';
+import axios from 'axios';
 
-const App = () => {
-  const [page, setPage] = useState('profiles')
-  const [profile, setProfile] = useState()
+import AstroProfiles from './components/AstroProfiles/AstroProfilePage'
+import Matches from './components/MatchList/Matches'
+import {HomeContainer , ButtonContainer , IconsApp , Logo} from './styled'
+
+import matches from './assets/metchesicon.png'
+import reset from './assets/reset.png'
+import logo from './assets/astromatchelogo.png'
+
+function App() {
   
-  //change page to match
+  const  [ page, setPage ] = useState('profiles')
+
   const changePage = () => {
-    if (page === "profiles"){
-      setPage("matches")
+    if (page === 'profiles') {
+      setPage('matches')
+    } else {
+      setPage('profiles')
     }
-    else {
-      setPage("profiles")
-    }
-  } 
-
-  const getProfile = () => {
+  }
+  
+  const [ profile, setProfile ] = useState()
```

Comment:
Voce precisa colocar um valor inicial! Aqui poderia ser um objeto vazio.
```suggestion
  const [ profile, setProfile ] = useState( { } )
```